tests/lexer/tstrlits.nim

```diff
@@ -1,19 +0,0 @@
-discard """
-  output: "a\"\"long string\"\"\"\"\"abc\"def_'2'●𝌆𝌆A"
-"""
-# Test the new different string literals
```

Comment:
why was this file removed?

Reply:
An oversight.

Reply:
=> https://github.com/nim-lang/Nim/pull/17500